app/src/main/java/ch/epfl/favo/favor/Favor.java

```diff
@@ -197,12 +200,17 @@ public void setReward(double reward) {
   }
 
   public void setAccepterId(String id) {
```

Comment:
Each favor maintain a list of Ids to record committed users except the first one in list is Requester. Each time one helper commits a favor, he appends his userId to the list, but doesn't change statusId. If a requester finally choose a helper as the accepter, he firstly reset/clear the list of committed helpers, then only puts the accepterId in the list.

Reply:
Im still not sure if I follow i find this slightly confusing, is there any way we can maybe simplify this, or change it so that the requester is in its own field instead of in an array where we are just supposed to know the first is the requester, is there any type checking enforcing that the first will always be the requester?

Reply:
I see your point. Well, this is mainly designed for the ease of querying user's participated favors, either requested or accepted/commited. So we can fetch favors by single time of query using array-contains in Firebase.
Alternative scenario is to have 3 separated fields: String requesterId, String accepterId, List participatedId.
This change will need delete all entries in database, so I prefer to discuss this scenario during tomorrow meeting.